experimental/howMuchLumiInNtuples.py

```diff
@@ -0,0 +1,34 @@
+import ROOT 
+from ROOT import gROOT, TFile, TH1F
+import json
+import glob
+
+inputFiles = glob.glob('/hdfs/TopQuarkGroup/phxlk/ntuple/v0.0.8/Spring16/SingleMuon_Run2016E_PromptReco_v2/tmp/*/*.root')
```

Comment:
could you please make this into an argument?
Since this is the only dynamic thing in the script, something as simple as
```python
import sys
inputFiles = sys.argv[1:]
```
should be enough. The Linux shell will automatically evaluate expressions like `<path>/*.root` so glob is also not needed